### DISCUSSION TITLE
Changes in taxation method

### DISCUSSION THREAD

**Discussion by nothingmuch1** [phplime/Qrexorder#13] [Ideas]
Hi Nazmul,

Hope I am not bothering you with my requests, but I love your work, and hence these requests. So, right now, I want to request you to introduce some changes to the way taxes are calculated.

**Current method**
Currently, only one tax can be applied, but in reality, restaurant owners need to charge multiple taxes, such as in India, CGST and GST, and if it liquor, then extra VAT is added. I am adding an image for your reference.
![image](https://github.com/phplime/Qrexorder/assets/157798001/be8033e8-a810-47b3-8f9b-e1051637dc92)

**Changes needed**
1. Taxes should be calculated on net amount, and multiple tax feature. Such as 
 - Item Price: 100
 - Discount: 10
 - Tax 1: 10%
 - Tax 2: 10:
 Price to be paid: 98

Here is an image for your reference, and this is the way the final bill should look like
![image](https://github.com/phplime/Qrexorder/assets/157798001/7301610c-947d-4e19-a734-85651f5cfbc8)

**Here is how I think it can be done**
1. You can add a new feature in the settings, with options to add multiple taxes with their names
Here is a graphic of how it will look like
![image](https://github.com/phplime/Qrexorder/assets/157798001/c13845e8-d25d-4358-9aba-af478141f7e6)

2. While while adding an item, the restaurant may enable that particular tax and set a percentage for it.
Here is a graphic for your reference.
![image](https://github.com/phplime/Qrexorder/assets/157798001/5aa88631-3ae7-4a12-a050-6a9de992b7b6)

3. Now the checkout calculation may look like this.
![image](https://github.com/phplime/Qrexorder/assets/157798001/30b2ad44-99fa-477a-9113-2a5a761bca1f)

So if you can see now, the taxes are calculated on net amount after discounts, also multiple taxes can be added to the same cart.

Please add it in the next update. You may reply to this comment if my reqeust is not clear or if I was unable to explain properly.

Thanks and regards.
Wishig you best of luck.

**Comment by phplime** [comment]
Noted

**Reply by nothingmuch1** [reply]
Thanks bro........... waiting for it
